client/save-payment/html/src/recommended/app.js

```diff
@@ -0,0 +1,91 @@
+async function onPayPalLoaded() {
+  try {
+    const clientToken = await getBrowserSafeClientToken();
+    const sdkInstance = await window.paypal.createInstance({
+      clientToken,
+      components: ["paypal-payments"],
+      pageType: "checkout",
+    });
+
+    const paymentMethods = await sdkInstance.findEligibleMethods({
+      currency: "USD",
+    });
+
+    if (paymentMethods.isEligible("paypal")) {
+      setupPayPalButton(sdkInstance);
+    }
+  } catch (error) {
+    console.error(error);
+  }
+}
+
+const paymentSessionOptions = {
+  async onApprove(data) {
+    console.log("onApprove", data);
```

Comment:
Should we create a payment token in `onApprove()` similar to this doc? Ex:
```ts
onApprove: async ({ vaultSetupToken }) => {
            return fetch("example.com/create/payment/token", { body: JSON.stringify({ vaultSetupToken }) })
        },
```


https://developer.paypal.com/docs/checkout/save-payment-methods/purchase-later/js-sdk/paypal/#integrate-back-end